src/vs/workbench/contrib/terminal/browser/xterm/xtermTerminal.ts

```diff
@@ -303,6 +303,8 @@ export class XtermTerminal extends DisposableStore implements IXtermTerminal, II
 		this._updateTheme();
 		if (!this._container) {
 			this.raw.open(container);
+			// HACK: This must be done after the viewport is created
+			this.raw.options.screenReaderMode = this._configHelper.config.accessibleBufferContentEditable !== 'off';
```

Comment:
I was almost going to remove screenReaderMode completely, but we don't want to do the live region processing when accessibility mode is off. I think in xterm.js we should pull the accessible buffer into its own file and always enable it since it's lazily processed and wouldn't impact performance, just add functionality.

Reply:
As is would be a pretty big perf regression in terminal throughput, so I think that needs to happen first

Reply:
makes sense, just wanted to avoid a large PR if possible